config.js

```diff
@@ -1,22 +1,20 @@
 System.config({
-  "baseURL": "/rights-inspector",
-  "transpiler": "traceur",
-  "paths": {
-    "*": "*.js",
-    "github:*": "jspm_packages/github/*.js",
-    "npm:*": "jspm_packages/npm/*.js"
-  }
-});
+  //baseURL: '/rights-inspector',
```

Comment:
Yea - not great, but better than commenting, committing, remembering to uncomment, uncomment, commit.